pkg/platform/kube/platform.go

```diff
@@ -168,7 +168,7 @@ func (p *Platform) CreateFunction(createFunctionOptions *platform.CreateFunction
 			FunctionStatus: &functionconfig.Status{
 				HTTPPort: defaultHTTPPort,
 				State:    functionconfig.FunctionStateError,
-				Message:  briefErrorMessage,
+				Message:  briefErrorsLog,
```

Comment:
Sticking with message is more consistent here, even though arguably not the best field name to start with